File: D309.pas
program D309;

var S, T :string[100];  i : integer;

begin
readln(S);
readln(T);
for i := 1 to length(s) do if ord(s[i]) > 96 then s[i] := chr(ord(s[i])-32);
for i := 1 to length(t) do if ord(t[i]) > 96 then t[i] := chr(ord(t[i])-32);

if s = t then writeln('Equal') else
  if s < T then writeln('Smaller') else writeln('Greater');
  readln;
end.

File: D309.py
count = 0
a = list(input().lower())
b = list(input().lower())
if len(a) > len(b):
    print("Greater")
elif len(a) < len(b):
    print("Smaller")
else:
    for i in a:
        a[count] = ord(a[count])
        count = count + 1
    count = 0
    for k in b:
        b[count] = ord(b[count])
        count = count + 1
    count = 0
    while a[count] == b[count] and count < len(a):
        count = count + 1
    if a[count] < b[count]:
        print("Smaller")
    elif a[count] > b[count]:
        print("Greater")
    else:
        print("Equal")
